screenshots/src/main/java/com/appunite/loudius/screenshots/components/LoudiusFullScreenError.kt

```diff
@@ -56,11 +59,17 @@ fun ScreenErrorWithSpacers(
     onButtonClick: () -> Unit,
 ) {
     Column(
-        modifier = modifier.padding(32.dp).fillMaxSize(),
+        modifier = modifier
+            .background(color = Color.White)
```

Comment:
question: why you have you had to add background modifier?

Reply:
I will provide a link to a document(work in progress): 
https://www.notion.so/appunite/Experiment-Golden-tests-for-jetpack-compose-74e68882a08f4126b9b1d76d22f55706?pvs=4#5eb9d312a0f2435eb4d9241a1a976820

Basically on the screenshot this `LoudiusFullScreenError()` had no background before. So it looked bad. 
I couldn't find the reason why it is like that.